### PR TITLE
cleaning up remaining templates from #10918

### LINKED ISSUE #16294: tidy up MDN search field in devhub base templates that's no longer used
When the base templates were [cleaned up](https://github.com/mozilla/addons-server/pull/10926/files#diff-520fc674741702badfb6599cea9cce0f5a93e5bec543bf24caeb834d99989aa3L100 (edited) ) as part of the legacy frontend removal the search field was (knowingly) dropped from the devhub pages too.  But the code in the devhub base templates wasn't.  That should be removed.

### PR DESCRIPTION
**Pull request by ankushduacodes**
Fixes #16294

Changes Introduced:
-- Minor Template cleanup

### CODE REVIEW

**Comment by ankushduacodes** [comment]
@eviljeff One of the tests are failing I am not sure why. But can confirm that this same error about `local_settings` import came up when I was trying to setup the dev env locally.

Any insight?

**Review by eviljeff — APPROVED** [review]
thanks r+

File: src/olympia/lib/settings_base.py [hunk 1]
```diff
@@ -1286,10 +1286,6 @@ def get_db_config(environ_var, atomic_requests=True):
     ANALYTICS_HOST,
     PROD_CDN_HOST,
 )
-CSP_FORM_ACTION = (
-    "'self'",
-    'https://developer.mozilla.org',
```
Comment (eviljeff) [inline src/olympia/lib/settings_base.py:1291]: actually, can we change this to just drop the `https:// ....` line, and keep the rest of the tuple as now.

File: src/olympia/amo/tests/test_csp_headers.py [hunk 2]
```diff
@@ -97,7 +92,6 @@ def test_self_in_common_settings(self):
         assert "'self'" in base_settings.CSP_CONNECT_SRC
         assert "'self'" in base_settings.CSP_CHILD_SRC
         assert "'self'" in base_settings.CSP_FRAME_SRC
-        assert "'self'" in base_settings.CSP_FORM_ACTION
```
Comment (eviljeff) [inline src/olympia/amo/tests/test_csp_headers.py:100]: and revert this line too

**Review by eviljeff — CHANGES_REQUESTED** [review]
sorry, we looked further into how the dependency (https://django-csp.readthedocs.io/en/latest/configuration.html) uses these settings and it appears that `CSP_FORM_ACTION` _does_ need to be defined, even when it's `"'self'"`.

**Review by eviljeff — APPROVED** [review]
thanks! r+

**Pull request merged**